newIDE/app/src/EventsFunctionsList/index.js

```diff
@@ -36,8 +38,23 @@ export type EventsFunctionCreationParameters = {|
   name: ?string,
 |};
 
+const renderEventsFunctionLabel = (eventsFunction: gdEventsFunction) =>
+  eventsFunction.isPrivate() ? (
+    <>
+      <Tooltip title="This function won't be visible in the events editor">
```

Comment:
One thing I forgot to mention in the checklist was that I added a tooltip to the icon. The goal is to give more context about the icon. Though I am not really sure about the wording.  is `Events editor` a thing?

Reply:
Events editor or events sheet are synonym and right :)